src/azure-cli/HISTORY.rst

```diff
@@ -16,6 +16,10 @@ Release History
 * Upgrade azure-mgmt-storage version to 7.1.0
 * `az storage account create`: Add `--encryption-key-type-for-table` and `--encryption-key-type-for-queue` to support Table and Queue Encryption Service
 
+**SQL**
```

Comment:
2.0.80 is already released. Your release notes should be in the position above 2.0.80